### DISCUSSION TITLE
Nilan integration doesn't work with Home Assistant 2023.2.0

### DISCUSSION THREAD

**Discussion by salosa** [veista/nilan#43] [General]
From log: Error importing platform config_flow from integration nilan to set up nilan configuration entry: No module named 'pymodbus.client.sync'
I think pymodbus v3.1.1 doesn't any more support 'pymodbus.client.sync'

Simple fix is to replace module 'pymodbus.client.sync' with 'pymodbus.client.' in config_flow.py

**Comment by veista** [comment]
Yes, thanks for the fix. Though the underlying issue still remains. The version is not explicitly defined in the manifest.
